### ISSUE TITLE
Regression tests with some big angular apps

### ISSUE THREAD

**Issue by tilmanpotthof** [Gillespie59/eslint-plugin-angular#260]
Regarding a 1.0.0 release it would be great to have a set of big real world application which follow a lot of the plugin rules and can be used to verify regressions that may not be covered by our unit tests.

I can imagine different scenarios how this could work. Two examples...

1. Simple but manual

Before each release some of use build the plugin locally and run the check against applications we are building

Pros: 
* We can also check private apps we write for customers
* No big investment in a complex ci setup

Cons:
* Manual process (error prone!)

2. Integrated in Travis

We find some open source apps or angular modules which already use the plugin or at least follow a lot of rules without using it.
Then we somehow run a new release (or each build) automated against these projects.

Pros:
* Automated!!!

Cons:
* We have to find enough (or big enough) projects which cover a lot of rules

**Comment by remcohaszing** [comment]
Sounds cool. What about trying to convince the [angular-material](https://github.com/angular/material) team that it's a good idea to use (ESLint and) this plugin?

They seem to follow some of our rules (e.g. file names) and it's officially supported by the AngularJS team. They just haven't automated some of the checks yet. ;)

**Comment by tilmanpotthof** [comment]
Good idea. As they are not using ESLint so far, we should check there contribution guidelines carefully to not run into some bureaucratic problems ;-) I could do that or would you like to?

Any other project suggestions?

**Comment by alecxe** [comment]
Correct me if I am wrong. The main purpose for this integration test is to make sure the plugin just does not fail in an unpredicted manner - does not throw any "runtime" specific errors..since you cannot possibly know what violations are currently present, right? Thanks.
